core/system/src/privcall.c

```diff
@@ -15,7 +15,18 @@
  * limitations under the License.
  */
 #include "api/inc/privcall_exports.h"
+#include <uvisor.h>
```

Comment:
Adding it there didn't work out so well. I got errors from certain functions previously declared as non-static being declared as static and other things. We should include a platform-agnostic configuration.h instead, if possible.